app/views/create-secret.html

```diff
@@ -0,0 +1,38 @@
+<default-header class="top-header"></default-header>
+<div class="wrap no-sidebar">
+  <div class="sidebar-left collapse navbar-collapse navbar-collapse-2">
+    <navbar-utility-mobile></navbar-utility-mobile>
+  </div>
+  <div class="middle surface-shaded">
+    <!-- Middle section -->
+    <div class="middle-section">
+      <div class="middle-container">
+        <div class="middle-content">
+          <div class="container surface-shaded">
+            <div ng-if="!project">Loading...</div>
+            <div class="row" ng-if="project">
+              <div class="col-md-10 col-md-offset-1">
+                <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+                <alerts alerts="alerts"></alerts>
+                <div class="mar-top-xl">
+                  <h1>Create Secret</h1>
+                  <div>
+                    <span class="help-block">
```

Comment:
Nit: should be able to put `class="help-block"` on the div and get rid of the inner span